docs/search/views/create-search-views.md

```diff
@@ -24,3 +30,145 @@ Follow these instructions to create a new search view using the GDN console web
     - **Direction -** Set the sorting order to ascending (default) or descending.
 
 After creating a view, you can **Rename** or **Delete** it from the **Search** screen.
+
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to [Create a Search View](https://www.macrometa.com/docs/api#/operations/createView).
+
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+Create a view.
+
+```bash
+gdnsl view create <view-name> [flags]
+```
+
+### Examples
+
+```bash
+  # Create test view on demo collection
+  gdnsl view create test --collection demo
+
+  # Create test view on demo collection with analyzer
+  gdnsl view create test --collection demo --analyzer identity
+
+  # Create test view on demo collection with fields
+  gdnsl view update test --collection demo --field "name:text_en,text_fr" --field "age:text_en"
+  
+  # Create test view on demo collection with store-values
+  gdnsl view create test --collection demo --store-values none
+
+  # Create test view on demo collection with track-list-positions
+  gdnsl view create test --collection demo --track-list-positions
+
+  # Create test view on demo collection with include-all-fields
+  gdnsl view create test --collection demo --include-all-fields
+
```

Comment:
we should also add gdnsl example command with primary sort field..

gdnsl view create test --collection demo --field "name:text_en,text_fr" --field "age:text_en" --primary-sort-field "name:asc"

We should also explain here that name and age are the fields and text_en / text_fr are the analyzers being used and similarly that we are sorting based on field "name" in ascending order (asc).. and desc for descending

Reply:
I generally just link to the CLI topic or heading with the info needed, cuts down on the maintenance overhead. If this is wrong, though, we should also review the source CLI topic, @jamesm-macrometa .